apps/web/src/pages/auth/components/QuestionnaireForm.tsx

```diff
@@ -161,12 +174,6 @@ export function QuestionnaireForm() {
       <Controller
         name="domain"
         control={control}
-        rules={{
```

Comment:
Alternatively, we can add it as a custom rule and let it trigger as part of react-hook-form lifecycle. For example:

```
rules = {{
  validate: {
    isValidDomain: v => parse(v)...
  }
}}```

For more information PTAL https://www.react-hook-form.com/api/useform/register/#options:~:text=%7D%0A/%3E-,validate,-Function%20%7C%20Object

I am not sure what other benefits this gives us but If I had to guess it would make it simpler to trigger all validations at once on form click or maybe on blur if we want to.